src/index.ts

```diff
@@ -164,7 +171,8 @@ function newFactory(config: Config, interfaceDefaults: Record<string, string>, t
           } else if (fieldType instanceof GraphQLObjectType) {
             return `o.${f.name} = maybeNew${fieldType.name}(options.${f.name}, cache);`;
           } else if (fieldType instanceof GraphQLInterfaceType) {
-            const implTypeName = interfaceDefaults[fieldType.name] || fail();
+            // Default to the first type which happens to implement the interface
+            const implTypeName = interfaceImpls[fieldType.name][0] || fail();
```

Comment:
In theory this could now call `maybeNew${fieldType.name}`? Not sure that it matters, but I think that effectively this is doing what the new `maybeNew...` method is doing? I'd just inlined it out of worked-at-the-time-ness before.

Reply:
Yes, that is true. I'll see if I can adjust.

Reply:
I didn't have time to do this. Shall I file a ticket?

Reply:
I'm fine leaving as-is, we'll just keep a todo in our heads.